src/models/SDORdfConcept.py

```diff
@@ -76,7 +78,7 @@ def __init__(self, metadata, concept_type, profile, logger, cache):
         self.__parent_to_rdf(metadata)
 
     # TODO: make a module for rights
-    def rights_to_license_uri(self, payload=None):
+    def rights_to_license_uri(self, payload=None):  # noqa: C901 #TODO
```

Comment:
This comment is saying the function is too complex or can be written less error prone. Not necessarily wrong, but it appears it is better to refactor this a bit.

Reply:
agreed to do that in another PR